misc/null-resource-workflow-environment/env0.workflow.yml

```diff
@@ -1,8 +1,8 @@
 environments:
   first-alias:
     name: first-name
-    templateName: 'null-resource'
+    templateName: "null-resource-1"
```

Comment:
sorry, I know it's petty, but can you please use `'` instead of `"` like it is in the rest of the workflow templates?

Reply:
btw, where are those `null-resource-1` and `null-resource-2`?

Reply:
it was VS code auto pretty, haven't noticed that, thanks [2e50699](https://github.com/env0/templates/pull/158/commits/2e506994afbdc0b112c3b20be9f15ac63facc488)
and I will create those in the e2e test, currently we are creating only 1 template named `null-resource`, so I'll create 2 different ones, so the workspace name uniqueness won't hinder us